azure_costs_exporter/prometheus_collector.py

```diff
@@ -65,9 +66,15 @@ def _get_azure_data(self, month=None):
         url = "https://ea.azure.com/rest/{0}/usage-report?month={1}&type=detail&fmt=Json".format(self._enrollment,
                                                                                                  month)
 
-        rsp = requests.get(url, headers=headers, timeout=10)
-        rsp.raise_for_status()
+        try:
+            timeout = int(float(request.headers.get('Scrape-Timeout-Seconds', 10)))
+        except (RuntimeError, ValueError) as e:
+            # The RuntimeError is raised when we are not in a flask request
+            # context, for example when running the unit tests for the collector
+            timeout = 10
```

Comment:
Use a global constant `DEFAULT_SCRAPE_TIMEOUT = 10` in place of the two occurrences of `10`.